docs/src/add-services/influxdb.md

```diff
@@ -28,41 +28,36 @@ The format exposed in the ``$PLATFORM_RELATIONSHIPS`` [environment variable](../
 
 {{% endpoint-description type="influxdb" /%}}
 
-```php
-<?php
-// This assumes a fictional application with an array named $settings.
-if (getenv('PLATFORM_RELATIONSHIPS')) {
-	$relationships = json_decode(base64_decode($relationships), TRUE);
-
-	// For a relationship named 'influxtimedb' referring to one endpoint.
-	if (!empty($relationships['influxtimedb'])) {
-		foreach ($relationships['influxtimedb'] as $endpoint) {
-			$settings['influxdb_host'] = $endpoint['host'];
-			$settings['influxdb_port'] = $endpoint['port'];
-			break;
-		}
-	}
-}
-```
+{{< codetabs >}}
+
+---
+title=PHP
+file=static/files/fetch/examples/php/influxdb
+highlight=php
+---
+
+{{< /codetabs >}}
```

Comment:
:clipboard: This shouldn't be in codetabs if it's only one language. But since an example is also available for Node.js, that should be aded.